modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/internal/upgrade/v6_5_0/FriendlyURLUpgradeProcess.java

```diff
@@ -30,22 +30,23 @@ protected void doUpgrade() throws Exception {
 
 			try (PreparedStatement preparedStatement1 =
 					connection.prepareStatement(
-						"select categoryId, name from MBCategory order by " +
-							"name, categoryId asc");
+						"select categoryId, ctCollectionId, name from " +
```

Comment:
@adolfopa, Brian will most likely want a change of the order so that it matches service.xml: ctCollectionId -> categoryId -> name

Reply:
Updated to reflect service.xml order. Thanks!